index.html

```diff
@@ -55,8 +56,107 @@ <h1 class="hero-title">
           <img src="images/company-logos/microsoft-logo.png" alt="company logo">
           <img src="images/company-logos/walmart-logo.png" alt="company logo">
           <img src="images/company-logos/fedex-logo.png" alt="company logo">
-        </div>
-      </section>   
+          </div>
+      </section>
+      <!--feature-section-->
+      <section class="feature-section">
+        <div class="feature-section-full">
+         <!--feature-header-->
```

Comment:
- [ ] takeout the comments
> it's alright when you use them as guide within your code. However, do not push them.
 as a rule of thumb, indent your code to improve its readability.